uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -729,20 +729,126 @@ Feature: GGMQ-1
     And I set MQTT publish retain flag to true
 
     When I publish from "publisher" to "iot_data_0" with qos 0 and message "Hello world0"
-    And I set MQTT subscribe retain handling property to "MQTT5_RETAIN_SEND_AT_SUBSCRIPTION"
     When I subscribe "subscriber" to "iot_data_0" with qos 0
     And message "Hello world0" received on "subscriber" from "iot_data_0" topic within 5 seconds
 
+    And I set MQTT publish retain flag to false
+
     When I publish from "publisher" to "iot_data_1" with qos 0 and message "Hello world1"
-    And I set MQTT subscribe retain handling property to "MQTT5_RETAIN_SEND_AT_NEW_SUBSCRIPTION"
     When I subscribe "subscriber" to "iot_data_1" with qos 0
-    And message "Hello world1" received on "subscriber" from "iot_data_1" topic within 5 seconds
+    And message "Hello world1" received on "subscriber" from "iot_data_1" topic within 5 seconds is false expected
```

Comment:
what does "false expected" mean? These steps should read like regular English sentences.

Do you mean that the message is _not_ received?

Reply:
yes, the message is not received, should I change the step?

Reply:
Yes.

`And message "Hello world1" is not received on "subscriber" from "iot_data_1" topic within 5 seconds`

Reply:
With false/true sentence we can use macro in scenarios